### ISSUE TITLE
IntelliSense painful to use with directory containing multiple projects

### ISSUE THREAD

**Issue by suedama1756** [OmniSharp/omnisharp-vscode#640]
## Environment data

.NET Command Line Tools (1.0.0-preview2-003121)

Product Information:
 Version:            1.0.0-preview2-003121
 Commit SHA-1 hash:  1e9d529bc5

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.11
 OS Platform: Darwin
 RID:         osx.10.11-x64

VS Code version:

1.4.0

C# Extension version:

1.3.0

## Steps to reproduce

I have a simply folder structure with a global.json in the root referencing several child folders:

- global.json
- Project.One
   - project.json
- Project.Two
  - project.json

All projects created using dotnet new

## Expected  behavior

Intellisense to work, go to symbol to work, project.json files to be found.

## Actual behavior

The C# plugin does not provide intellisense at all if I select the root folder, I have to continuously switch between projects using the "omnisharp: select project" command to select an individual project.json, even then it often doesn't work. All non current projects show huge numbers of errors (e.g. type 'System.Object' could not be found). Go to symbol does not work at all.

I've tried creating new projects from scratch etc. same behaviour...

**Comment by suedama1756** [comment]
I belief this is related to: http://stackoverflow.com/questions/38670295/brew-refusing-to-link-openssl/38706045#38706045 as I have a crypto error in the omnisharp logs. I'm continuing to investigate how to fix this without downgrading openssl in brew. 

I've already run: sudo install_name_tool -add_rpath /usr/local/opt/openssl/lib /usr/local/share/dotnet/shared/Microsoft.NETCore.App/1.0.0/System.Security.Cryptography.Native.dylib

which fixed issues in dotnet command

**Comment by suedama1756** [comment]
I can confirm the crypto error on startup is fixed by running:

sudo install_name_tool -add_rpath /usr/local/opt/openssl/lib ~/.vscode/extensions/ms-vscode.csharp-1.3.0/out/.omnisharp-coreclr/System.Security.Cryptography.Native.dylib

**Comment by suedama1756** [comment]
Behaviour is much more reliable now, however, I still have to switch projects to get intellisense.

**Comment by suedama1756** [comment]
Once the debugger is downloaded you also need to run the same command on the debuggers version of System.Security.Cryptography.Native.dylib

```bash
install_name_tool -add_rpath /usr/local/opt/openssl/lib ~/.vscode/extensions/ms-vscode.csharp-1.3.0/coreclr-debug/debugAdapters/System.Security.Cryptography.Native.dylib
```

**Comment by scrudgey** [comment]
The extension is also not working for me. It fails on installing the .net core debugger.

**Comment by scrudgey** [comment]
Note: I resolved my issue with the .net core debugger not installing. I did both:

1. Downloaded and installed dotnet 1.0.0-preview2-003121 for mac os x from microsoft's website, and

2.sudo install_name_tool -add_rpath /usr/local/opt/openssl/lib /usr/local/share/dotnet/shared/Microsoft.NETCore.App/1.0.0/System.Security.Cryptography.Native.dylib

I am not sure if step 2 was necessary, as I did both before trying to launch VS Code again.
